pysetup/md_doc_paths.py

```diff
@@ -26,6 +27,7 @@
     EIP6800: DENEB,
     EIP7441: CAPELLA,
     EIP7805: FULU,
+    EIP7928: ELECTRA,
```

Comment:
I imagine clients will implement/test eip7928 from Fulu, not Electra. Should be pretty simple to bump now.